### ISSUE TITLE
DBH checks

### ISSUE THREAD

**Issue by ValentineHerr** [SCBI-ForestGEO/2023census#24]
We need to align our thresholds between the app and the GitHub actions, so @jess-shue will try to add the following statements to the app's DBH checks (on top of the absolute checks of -0.5cm for negative and 4cm for positive).

```
dbh_previous*0.75 > dbh_current & !is.na(dbh_previous) & !is.na(dbh_current) # suspiciousNegativeGrowth
dbh_previous*1.92 < dbh_current & !is.na(dbh_previous) & !is.na(dbh_current) # suspiciousPositiveGrowth
```

**Comment by teixeirak** [comment]
I completely agree that it's best to program checks into the app whenever possible. It's always easiest to catch the errors while they're at the tree.

I agree that reducing the measurement error will be a change from previous censuses, but I think it's purely a positive change. The question is whether it would become too burdensome for the crew to go back and check suspicious measurements. I think it will take a bit of trial to optimize the criteria for flagging potential errors, and there's definitely some philosophical calls there (what level of error is tolerable? what's the optimal tradeoff between error-free data and greater efficiency?).

**Comment by teixeirak** [comment]
I just spoke with Madeleine Udell (Stanford) about turning this issue into a project for data science MS students at Stanford. The idea would be to use unsupervised machine learning to detect detect outliers.
